src/components/ebay-section-title/examples/07-all-giant/template.marko

```diff
@@ -0,0 +1,18 @@
+<ebay-section-title size="giant">
+    <ebay-section-title-title href="https://www.ebay.com">Today’s Deals – All With Free Shipping</ebay-section-title-title>
```

Comment:
Why do we have two different links here? 
Again, adding it as an option on the top might be cleaner.

Reply:
I forgot to remove href on `title`. This was an earlier idea and currently doesn't do anything.

I will likely move to using attributes on `ebay-section-title` parent.

Reply:
done